pertpy/tools/_cinemaot.py

```diff
@@ -346,7 +346,7 @@ def get_dim(
         sk.fit(vm)
         wm = np.dot(np.dot(np.sqrt(sk._D1), vm), np.sqrt(sk._D2))
         u, s, vt = np.linalg.svd(wm)
-        dim = np.min(sum(s > (np.sqrt(data.shape[0]) + np.sqrt(data.shape[1]))), adata.obsm[use_rep].shape[1])
+        dim = min(sum(s > (np.sqrt(data.shape[0]) + np.sqrt(data.shape[1]))), adata.obsm[use_rep].shape[1])
```

Comment:
Hi, thank you so much for taking another look at the code!

I'm still unsure about the functionality of this code line. At least with the example I'm using, I get an error because `(s > (np.sqrt(data.shape[0]) + np.sqrt(data.shape[1])))` returns a boolean, but `sum` expects an iterable. It's possible that I'm not testing it with a suitable example - in that case, maybe you could provide me with one.
Many thanks in advance!

Reply:
s is a vector and (s > (np.sqrt(data.shape[0]) + np.sqrt(data.shape[1])) returns a boolean array. sum can work on that boolean array. Not sure what is the problem there... Did you set s as an array?

Reply:
Oh, I indeed didn't realize s is a vector - sorry, my bad! The method works for me too now.
In my opinion, we could merge the changes then. @Zethson, could you please have a final look?